src/main/java/bio/terra/workspace/service/crl/CrlService.java

```diff
@@ -0,0 +1,116 @@
+package bio.terra.workspace.service.crl;
+
+import bio.terra.cloudres.common.ClientConfig;
+import bio.terra.cloudres.common.cleanup.CleanupConfig;
+import bio.terra.cloudres.google.billing.CloudBillingClientCow;
+import bio.terra.cloudres.google.cloudresourcemanager.CloudResourceManagerCow;
+import bio.terra.cloudres.google.serviceusage.ServiceUsageCow;
+import bio.terra.workspace.app.configuration.external.CrlConfiguration;
+import bio.terra.workspace.service.crl.exception.CrlInternalException;
+import bio.terra.workspace.service.crl.exception.CrlNotInUseException;
+import bio.terra.workspace.service.crl.exception.CrlSecurityException;
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.auth.oauth2.ServiceAccountCredentials;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.time.Duration;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CrlService {
+  /** The client name required by CRL. */
+  public static final String CLIENT_NAME = "workspace";
```

Comment:
This constant is also used as the name of a resource in Sam. I don't think we should share the value for two different purposes. This used to be in the CrlConfiguration.java, but I moved it here, since it was the only place it was used.

In fact, neither of the statics are used outside of this module. I will make them private.